src/flownet/network_model/_generate_connections.py

```diff
@@ -309,6 +319,8 @@ def _create_entity_connection_matrix(
     aquifer_ends: List[Coordinate],
     max_distance_fraction: float,
     max_distance: float,
+    concave_hull_bounding_boxes: Optional[np.ndarray] = None,
+    n_non_reservoir_evaluation: Optional[int] = 10,
```

Comment:
When the number of points is set as default in the config file, will it ever not be defined?

Reply:
Unless you call the function via some other way.